packages/web-components/src/components/rux-tabs/rux-tab/rux-tab.scss

```diff
@@ -46,7 +49,8 @@
 }
 
 .rux-tab--small.rux-tab--selected {
-    border-bottom: 3px solid var(--color-border-interactive-default);
+    border-bottom: var(--border-width-lg) solid
+        var(--color-border-interactive-default);
```

Comment:
Non-Blocking Question: Did this break come from prettier?

Reply:
It must have. I swapped it back as it should have been. I've meant to ask if people have global prettier settings they like to use for uniformity.